examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -97,7 +97,11 @@
     "from dask_saturn import SaturnCluster\n",
```

Comment:
<p>what's the reason for this change? I'm mildly scared of going too far into the <code>pandas</code>  API, so when I see things in pandas that I've never seen before, I get a little nervous.</p><p> </p><p>If you tell me "james, <code>.columns.str</code>  is totally a thing and has been forever" then I'm fine with this. If you tell me "oh yeah this is new in <code>pandas</code>  1.0" then I think we should revert it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p><a href="https://stackoverflow.com/a/38931854" target="_blank">this SO answer</a> is from 2016 so I think its been around for a while. I did this mostly for consistency too because notebooks in other places had done it. I also have a tendency to try to avoid loops/list comprehensions as much as possible in pandas too so my default mode is to find ways around that</p>

Reply:
ok no prob